dev/SapphireTestReporter.php

```diff
@@ -1,6 +1,10 @@
 <?php
-if(!class_exists('PHPUnit_Framework_TestResult', false)) require_once 'PHPUnit/Framework/TestResult.php';
-if(!class_exists('PHPUnit_Framework_TestListener', false)) require_once 'PHPUnit/Framework/TestListener.php';
+if(!class_exists('PHPUnit_Framework_TestResult', false)) {
```

Comment:
It could have been except:

1. PHPUnit doesn't support PEAR installs any more; if it's installed globally as a phar then I doubt we can extend it like this anyway, which means...
1. It'll be installed by composer to get this to work anyway
1. This is master, we can break what we want